addons/2d-color-picker/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "2D color picker",
-  "description": "Replaces saturation and brightness sliders with a 2D color picker. Hold Shift while dragging the cursor to change the values on a single axis.",
+  "description": "Replaces saturation and brightness sliders in the paint editor with a 2D color picker. Hold Shift while dragging the cursor to change the values on a single axis.",
```

Comment:
This addon might support all color pickers someday, so I don't think it makes sense to change this

Reply:
Until it supports all color pickers, it should be clear that it doesn't. When it does, it should at least say "in color pickers", otherwise what "saturation and brightness sliders" are isn't obvious.